QUARKUS-6261.MD

```diff
@@ -0,0 +1,120 @@
+# QUARKUS-6261 - Upgrade to Hibernate ORM 7, Hibernate Reactive 3 and Hibernate Search 8
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6261
+
+Upstream issues:
+
+- https://github.com/quarkusio/quarkus/issues/42247
+- https://github.com/quarkusio/quarkus/issues/38948
+- https://github.com/quarkusio/quarkus/issues/43368 
+- https://github.com/quarkusio/quarkus/issues/45164
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/41310
+
+Quarkus Hibernate extensions now uses newer version of underlying Hibernate libraries, which brings improvements and breaking changes we need to cover.
+The upstream PR is marked as breaking and noteworthy and breaking changes are recorder in the Quarkus migration guide, therefore the breaking changes are acceptable.
+
+**Hibernate ORM 7.0 and Jakarta Persistence 3.2** (product status: supported):
```

Comment:
This section is hard to read, because it mixes:
1. Descriptions of changes
2. Opinions about these changes (eg `Suggestion in the migration guide seems to be racy at best`)
3. Suggested actions
Could you reorganize this a bit? Probably, move some things to the "Scope" and create issues upstream for others

Reply:
> Suggestion in the migration guide seems to be racy at best

btw that's common sense, I won't report it as an issue because it must be clear to everyone, what's to report :-), I'll just drop that opinion.